shogun-lib/src/main/java/de/terrestris/shoguncore/model/Application.java

```diff
@@ -41,17 +44,15 @@
     @Type(type = "jsonb")
     @Column(columnDefinition = "jsonb")
     @Basic(fetch = FetchType.LAZY)
-    private Map<String, Object> layerTree;
+    private LayerTree layerTree;
 
     @Type(type = "jsonb")
     @Column(columnDefinition = "jsonb")
     @Basic(fetch = FetchType.LAZY)
-    private Map<String, Object> layerConfig;
+    private LayerConfig layerConfig;
 
     @Type(type = "jsonb")
     @Column(columnDefinition = "jsonb")
     @Basic(fetch = FetchType.LAZY)
-    private Map<String, Object> toolConfig;
-
+    private LayerToolConfig toolConfig;
```

Comment:
```suggestion
    private ApplicationToolConfig toolConfig;
```

Reply:
I prefixed all types that reside inside Application with `Application`